e2e/rdctl.e2e.spec.ts

```diff
@@ -695,6 +695,24 @@ test.describe('Command server', () => {
     });
 
     test.describe('set', () => {
+      const unsupportedPrefsByPlatform = {
```

Comment:
Consider typing this more explicitly so it's easier to reason about:
```suggestion
      const unsupportedPrefsByPlatform: {[x in NodeJS.Platform]?: [string, any][]} = {
```

This means line 714 can become:
```ts
      const unsupportedOptions = unsupportedPrefsByPlatform[process.platform] ?? [];
```

Reply:
ok